lib/vrf.h

```diff
@@ -159,18 +159,6 @@ static inline int vrf_is_user_cfged(struct vrf *vrf)
 	return vrf && CHECK_FLAG(vrf->status, VRF_CONFIGURED);
 }
 
-/* Mark that VRF has user configuration */
```

Comment:
so the idea is that these public apis were ... mis-used, so there shouldn't be public apis?